content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst

```diff
@@ -1,54 +1,245 @@
+   <inventory/routes/strategies/putaway/wrong-location>`.
+
+Use putaway rules
+-----------------
+
+Once a putaway rule is configured, the product it specifies is automatically routed to the
+:guilabel:`Store to` location upon arriving in the :guilabel:`When product arrives in` location.
+
+For example, on a putaway rule with the :guilabel:`When product arrives in` location set to
+`WH/Stock`, internal stock moves are automatically created when products first enter the warehouse.
+To trigger this putaway rule, begin by creating a :ref:`purchase order
+<inventory/purchase/manage_deals/manage>` (PO) to bring products into the warehouse.
 
-Now, if I purchase apples and carrots to my supplier, they will be grouped in the same receipt but
-redirected to the right location automatically, thanks to putaway rules. This information is
-available from *Inventory Report*, under the reporting menu.
+To create a :guilabel:`Request for Quotation` (RFQ), go to the :menuselection:`Purchase` app, and
+click :menuselection:`Create`. Then, add products to the :abbr:`RFQ (Request for Quotation)` by
+clicking :menuselection:`Add a line`, and selecting the desired product(s) from the drop-down menu
+in the :guilabel:`Product` column.
 
-.. image:: putaway/putaw3.png
+Once ready, click :guilabel:`Confirm`. Doing so turns the :abbr:`RFQ (Request for Quotation)` into a
+:abbr:`PO (Purchase Order)`. Next, click :guilabel:`Send by Email` to send the :abbr:`PO (Purchase
+Order)` to the vendor if needed.
+
+.. image:: putaway/purchase-apples-and-carrots.png
    :align: center
+   :alt: View purchase order for apples and carrots.
+
+Begin receiving the incoming shipment by selecting the :guilabel:`Receipt` smart button on the
+:abbr:`PO (Purchase Order)`. Click :menuselection:`Receive products --> Validate` to move products
+into the warehouse. Since the putaway rule is set to trigger when products arrive in stock, internal
+transfers are automatically created when products arrive in the :guilabel:`Destination Location`,
+`WH/Stock`. This ensures products entering the warehouse are stored in specific locations suitable
+for their storage needs.
 
-.. image:: putaway/putaw4.png
+.. image:: putaway/receive-incoming-stock.png
    :align: center
+   :alt: In the WH reception operation, receive package of apples and carrots.
+
+Verify stock moves
+==================
+
+Confirm whether the putaway rule worked as intended and that the products have been moved to the
+correct location using the :guilabel:`Inventory Report`. View all products in stock in
+:menuselection:`Inventory app --> Reporting --> Inventory Report`. By default, the products are
+grouped by location.
+
+Select a collapsable product drop-down menu, which displays the current locations of the products.
+Then, select a location drop-down menu to view a list of all stock movements in and out of the
+location. Each of these lines are records of a stock move, called a *stock move line*.
```

Comment:
I'm thinking we should add :guilables: here for the active instruction - @StraubCreative what are your thoughts here?
```suggestion
Select a :guilabel:`collapsable product` drop-down menu, which displays the current locations of the
products. Then, select a :guilabel:`location` drop-down menu to view a list of all stock movements
in and out of the location. Each of these lines are records of a stock move, called a *stock move
line*.
```